src/detector.c

```diff
@@ -312,7 +312,9 @@ void train_detector(char *datacfg, char *cfgfile, char *weightfile, int *gpus, i
             else fprintf(stderr, "\n Tensor Cores are used.\n");
             fflush(stderr);
         }
-        printf("\n %d: %f, %f avg loss, %f rate, %lf seconds, %d images, %f hours left\n", iteration, loss, avg_loss, get_current_rate(net), (what_time_is_it_now() - time), iteration*imgs, avg_time);
+        fprintf(stdout, "\n{ uuid: '%s', training: { iteration: %d, loss: %f, avg_loss: %f, rate: %f, seconds: %lf, eta: %f } }\n", 
```

Comment:
I assume this JSON will be written to disk in the future

Reply:
we are running `darknet` with redirection to make a log file. 
and the log will be sent to the server by using the `file beats` of ELK.